build.sbt

```diff
@@ -113,7 +108,7 @@ lazy val extras = project
   .settings(sonatypeSettings: _*)
   .settings(
     name := "sbt-jmh-extras",
+    scalaVersion := "2.12.2",
```

Comment:
We're only benchmarking on JDK8+ now. But `sbt-jmh-extras` doesn't actually depend on Scala (see setting below this line), so I'm not sure if this is needed at all.